policyengine_us/variables/gov/states/al/tax/income/exemptions/retirement/al_retirement_exemption_person.py

```diff
@@ -0,0 +1,17 @@
+from policyengine_us.model_api import *
+
+
+class al_retirement_exemption_person(Variable):
+    value_type = float
+    entity = Person
+    label = "Alabama retirement exemption for each person"
+    unit = USD
+    # The Code of Alabama Section 40-18-19 (a)(13).
+    documentation = "https://casetext.com/statute/code-of-alabama/title-40-revenue-and-taxation/chapter-18-income-taxes/article-1-general-provisions/section-40-18-19-exemptions-generally"
+    definition_period = YEAR
+    defined_for = "al_retirement_exemption_eligible_person"
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.al.tax.income.exemptions.retirement
+        pension_income = person("taxable_pension_income", period)
```

Comment:
we need more income types, including:
```
    - taxable_ira_distributions
    - taxable_401k_distributions
    - taxable_sep_distributions
    - taxable_403b_distributions
    - keogh_distributions # Keogh plans don't have a Roth option.
    - pension_income
```
Then add this to the list of deductions.yaml

Reply:
use an add function 
add(person, period, ["taxable_ira_distributions", "taxable_401k_distributions"....]

Reply:
when adding to the list of deduction check if it is only a 2023 or did it exist before (the deduction)